sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -78,6 +83,11 @@ extensions:
         origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
         rpm -V "${origin_package}"
     - type: "script"
+      title: "copy openshift images from docker storage to CRI-O storage"
+      timeout: 10800
+      script: |-
+              for i in $(docker images --filter 'reference=openshift/origin-*' --format '{{.Repository}}:{{.Tag}}'); do sudo skopeo copy docker-daemon:$i containers-storage:$i; done
```

Comment:
reduce the whitespace, also can you break this into separate lines for better readability?
```sh
for i in $(docker images --filter 'reference=openshift/origin-*' --format '{{.Repository}}:{{.Tag}}'); do
 sudo skopeo copy docker-daemon:$i containers-storage:$i
done
```

Reply:
fixed